build/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Mount the project as a read only filesystem to /app and pass
+# along any arugments that were provided to this script
+docker run --volume "$(pwd):/app:ro" backus/rubocop-rspec "$@"
```

Comment:
Do you want the container to exist after running it?
Otherwise specify `docker run --rm --volume "$(pwd):/app:ro" backus/rubocop-rspec "$@"`